Code/GraphMol/FileParsers/PDBWriter.cpp

```diff
@@ -7,24 +7,28 @@
 //  which is included in the file license.txt, found at the root
 //  of the RDKit source tree.
 //
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <map>
 #include <string>
 #include <vector>
-#include <map>
-#include <iostream>
-#include <fstream>
-#include <algorithm>
+
+#ifdef RDK_BUILD_MAEPARSER_SUPPORT
```

Comment:
Doesn't seem like the PDBWriter should need to include stuff from maeparser

Reply:
Or moving the `MaeWriter` class to a separate header, that should work too.